### PR TITLE
chore: release v2.3.0

### PR DESCRIPTION
**Pull request by CallumGrindle**
Release v2.3.0.

This release includes:
- updates reactive deployment, 
- renames classes and deployment events to use correct LSP standard names 
- adds LSP3 avatar support
- adds generic contract options for digital assets
- Smart contract upgrade to v0.6.0
- Integrated changes in LSP2 mapping values

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,27 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [2.3.0](https://github.com/lukso-network/tools-lsp-factory/compare/v2.1.0...v2.3.0) (2022-05-30)
```
Comment (Hugoo) [inline CHANGELOG.md:5]: it takes from 2.1.0 instead of 2.2.0

File: docs/classes/lsp8-identifiable-digital-asset.md [hunk 2]
```diff
@@ -104,18 +103,22 @@ await lspFactory.LSP8IdentifiableDigitalAsset.deploy(
...
+    onDeployEvents: {
+      next: (deploymentEvent) => {
+        console.log(deploymentEvent);
+      },
+      error: (error) => {
+        console.error(error);
+      },
+      complete: (digitalAsset) => {
+        console.log('Deployment Complete');
+        console.log(digitalAsset);
```
Comment (frozeman) [inline docs/classes/lsp8-identifiable-digital-asset.md:116]: This should also be a contracts object with `LSP7DigitalAsset` as property, to be consistent across all deploy methods

Comment (frozeman) [reply docs/classes/lsp8-identifiable-digital-asset.md:116]: and it seems it is this way, so better lets call it `contracts` and log `contracts.LSP8....` to make it obvious

File: CHANGELOG.md (hunk 1 above)
Comment (CallumGrindle) [reply CHANGELOG.md:5]: Nice spot. I already manually removed the entries from 2.2.0 so its correct. But need to look into why it happened though - I think something to do with the squash merge somehow

**Pull request merged**